recipe/meta.yaml

```
@@ -76,11 +84,19 @@ outputs:
         - pip
         - pybind11
         - python
-        - setuptools <=64.0.2 # version requirement from cvxpy
+        - setuptools
       run:
         - ninja
         - numpy
         - python
```

It's nice that the import test exposes this 🙂 It seems `pennylane-core` should be a run requirement for `pennylane-lightning-core`?
```suggestion
        - python
        - {{ pin_subpackage('pennylane-core') }}
```

---

Yes, and it'd use the `pennylane-core` version seen from the same recipe as the minimal version, so in this case it's equivalent to
```
- pennylane-core >=0.29.0,<1
```
If you don't feel comfortable about `max_pin`, you can adjust it, the doc is here:
https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2-functions